### ISSUE TITLE
Failure occurred in prepare()

### ISSUE THREAD

**Issue by danielgarciagarcia** [yurikoles-aur/linux-drm-next-git#3]
Updating the package (version **6.2.r1156452.54ceb92724a8-1**) throws an error in the `prepare()` phase:
```

:: Checking for conflicts...
:: Checking for inner conflicts...
[Aur:1]  linux-drm-next-git-6.2.r1156403.a48bba98380c-2

  1 linux-drm-next-git               (Installed) (Build Files Exist)
==> Packages to cleanBuild?
==> [N]one [A]ll [Ab]ort [I]nstalled [No]tInstalled or (1 2 3, 1-3, ^4)
==> 
:: PKGBUILD up to date, Skipping (1/0): linux-drm-next-git
  1 linux-drm-next-git               (Installed) (Build Files Exist)
==> Diffs to show?
==> [N]one [A]ll [Ab]ort [I]nstalled [No]tInstalled or (1 2 3, 1-3, ^4)
==> 
:: (1/1) Parsing SRCINFO: linux-drm-next-git
==> Making package: linux-drm-next-git 6.2.r1156403.a48bba98380c-2 (mar 21 mar 2023 18:04:07)
==> Retrieving sources...
  -> Updating linux-drm-next-git git repo...
  -> Found config
==> Validating source files with sha256sums...
    linux-drm-next-git ... Skipped
    config ... Passed
==> Making package: linux-drm-next-git 6.2.r1156403.a48bba98380c-2 (mar 21 mar 2023 18:04:10)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Updating linux-drm-next-git git repo...
  -> Found config
==> Validating source files with sha256sums...
    linux-drm-next-git ... Skipped
    config ... Passed
==> Removing existing $srcdir/ directory...
==> Extracting sources...
  -> Creating working copy of linux-drm-next-git git repo...
Cloning into 'linux-drm-next-git'...
done.
Updating files: 100% (79623/79623), done.
Switched to a new branch 'makepkg'
==> Starting prepare()...
Setting version...
Usage: scripts/setlocalversion [srctree]
==> ERROR: A failure occurred in prepare().
    Aborting...
 -> error making: linux-drm-next-git
```

**Comment by danielgarciagarcia** [comment]
Fixed with commit https://github.com/yurikoles-aur/linux-drm-next-git/commit/572c790e1d397f8738881642f83c921768d0d2a5

**Comment by yurikoles** [comment]
@danielgarciagarcia, thank you for the report, and sorry for not so welcome self-service. The fix for `prepare()` was straight-forward, but I didn't want to push update until I figured out how to [fix a BTF-related error](https://github.com/yurikoles-aur/linux-drm-next-git/blob/572c790e1d397f8738881642f83c921768d0d2a5/config#L10996) during linkage of kernel. I decided that to fail fast is better than in the middle of the build.
